src/widgets/grid.ts

```diff
@@ -1,9 +1,15 @@
+import { ElementArrayFinder, ElementFinder } from "../wdio";
 import { ContextMenu } from './context-menu';
 import { Widget } from './widget';
-import { ElementArrayFinder, ElementFinder } from "../wdio";
 
 
 export class Grid extends Widget {
+    public async waitUntilLoaded(cssTag: string = ".ag-overlay-loading-wrapper", timeout: number = 5000) {
```

Comment:
Sorry for being late on this, but I would rename the argument from `cssTag` to `loadingCSSSelector`

Reply:
@EduAlberdi , one further question:
we saw recently that this tag can be already present, but containing the ephemeral "Loading" popup that is shown briefly. Then, "Loading" disappears and the data rows are displayed.
Can be consider that waitUntilLoaded() waits actually until the data rows are displayed? Or is it possible that it just waits until "Loading" popup is displayed?

Reply:
This was my first working example/implementation.
Maybe we can discuss in further meets when we have already tried it in our projects.